### DISCUSSION TITLE
[Enhancement] LastFM Changes

### DISCUSSION THREAD

**Discussion by coredev-uk** [ciderapp/Cider#668] [Feature Request]
> it only gets scrobbled when you pass onto the next song or close the app, until then it appears as now playing

![image](https://user-images.githubusercontent.com/64542347/148834613-227d4cb6-8809-4cea-8e02-2a80a0fe45b3.png)

**Comment by tva2000hn** [comment]
https://github.com/ciderapp/Cider/commit/f71a22b8e9a091352b76486836effe5d3b2b250b

**Comment by brotherelric** [comment]
Any ETA?

**Comment by tva2000hn** [comment]
Its fixed i think

**Comment by gustavocubas** [comment]
It's much better than it was before, but still a bit buggy. The big thing still causing problems for me is that if you pause a 5 minute song at, let's say, the 1 minute mark and never finish listening to it, the scrobble will still show up after the % delay is reached -- even if you didn't listen to it.

**Comment by coredev-uk** [comment]
I begun converting over to the newer lastfm module, its taking longer than expected.
